static/docs/commands-reference/pull.md

```diff
@@ -1,6 +1,6 @@
 # pull
 
-Pulls data files to the local workspace from a remote workspace.
+Pulls data files to the local workspace from a remote DVC cache based on relative changes between the two.
```

Comment:
The same with remote, see below. Also I don't understand this phrase "based on relative changes between the two". What is actually happening - it analyzes _all_ DVC files in the workspace, gets checksums, downloads missing files from the remote storage to the local cache and then runs `dvc checkout` to create links between cache and workspace.